int64 timestamp
float64 latitude
float64 longitude
float64 altitude
int32 NSV1
int32 NSV2